app/jenkins-for-jira-ui/src/components/icons/PluginIcon.tsx

```diff
@@ -0,0 +1,154 @@
+import React from 'react';
+import { cx } from '@emotion/css';
+
+type PluginIconProps = {
+	containerClassName: string,
+	svgClassName: string
+};
+
+export function PluginIcon({ containerClassName, svgClassName }: PluginIconProps): JSX.Element {
```

Comment:
Most of our svgs are set up as components but for some reason the PluginIcon and a couple of others are imported directly as svgs. Figured I'd updated this one but didn't before changing the import in JenkinsServerList as we'll eventually be ripping that out.